README.md

```diff
@@ -21,6 +21,23 @@ configuration is provided for running Nexus.
 For more information on the use of Vagrant, please see:
 https://github.com/GPII/qi-development-environments
 
+Running Nexus in a Docker container
+-----------------------------------
+
+You can also run the Nexus in a [Docker](https://docs.docker.com/get-docker) container.
+
+Once you have Docker installed, run the following commands to build a Docker image and start a container:
+
+* Build the image: `docker build -t nexus .`
+* Run the container: `docker run --name nexus -p 9081:9081 nexus`
+```
```

Comment:
Looks like this got left over after some editing.

Reply:
Thanks, fixed!